### ISSUE TITLE
Redundant repository?

### ISSUE THREAD

**Issue by raysarebest** [EastRobotics/GeneralRoboticsLibrary#1]
Based on the name, I'm guessing that the intent of this is to supply common code for future programmers to use. This is also the goal of [Lightning](https://github.com/EastRobotics/Lightning). I've been building Lightning over the past couple years, and though it still might not look like much (I have a lot of work still local to my personal box), I feel enough work has been done on it to allow it to be the definitive commons repository for the club, with any additions submitted via issue or pull request

**Comment by Bogidon** [comment]
@jcgrif has been developing a library with the kids at East since states. I'd love to hear about the goals of this project and its current progress.

@raysarebest is your code in a state where you can push it upstream? I'd like to see what you've been working on.

**Comment by jcgrif** [comment]
The idea would be to have a separate library that outlines the simplest use of the particular sensors, and useful routines.  Also, a place to collaborate on more complex behaviors that might encompass a number of sensors or routines.  This way each of the programmers can have access to tested routines that are known to work and instructions on the physical layout of the sensor. 

This is not a replacement for what you have done, but more of a learning tool and possibly a jumping off point to experiment with various sensors and behaviors.

**Comment by jcgrif** [comment]
This isn't really a library in the sense of programming construct, this is more in the sense of the traditional concept of a place where you go to find knowledge.

**Comment by raysarebest** [comment]
@Bogidon Getting there. I actually have some free time and ideas tonight, so I think I'm going to put in some more work to it. I've been greatly halted over the past few months because of the idiosyncrasies of ROBOTC and a killer bug I found in the preprocessor that Robomatter hasn't fixed in the 8 months since I've reported it, for some reason

@jcgrif Ah, I get it. I'll probably end up using this greatly as a sandbox for Lightning, then. I also have a feeling it'll be greatly a mirror of jpearman's work on the ROBOTC forum
